README.md

```diff
@@ -1,75 +1,249 @@
 ### Prerequisites
 
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+ linters
+ code editor
+ liveserver
+ images
+
+
 ### Setup
 
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+git@github.com:PrinceBadru/mobilefirst.git
 ### Install
 
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+ cd my-project
+  gem install
 ### Usage
 
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+```sh
+  rails server
+```
+
 ### Run tests
 
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+```sh
+  bin/rails test test/models/article_test.rb
+```
 ### Deployment
 
+You can deploy this project using:
 
+<!--
+Example:
 
-## Authors
+```sh
+
+```
+ -->
+Github pages.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
 
 👤 **Author1**
 
-- GitHub: [@PrinceBadru](https://github.com/PrinceBadru)
-- Twitter: [@NdawulaBadru](https://twitter.com/BadruNdawula)
 
+👤 **BADRU NDAWULA**
+- GitHub: [@badru](https://github.com/PrinceBadru)
+- Twitter: [@badru](https://twitter.com/BadruNdawula)
+- Linkedin: [@badru](https://www.linkedin.com/in/badru-ndawula-930b7b217/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🤝 Contributing
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[header]**
+- [ ] **[mobileview]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Give a ⭐️ if you like this project!
+<!-- ACKNOWLEDGEMENTS -->
 
-## Acknowledgments
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-- ### Original design idea by **Cindy Shin in Behance**, in association with the Creative Commons [license](./CC.md) of the design.
+> Give credit to everyone who inspired your codebase.
 
-## 📝 License
+I would like to thank...
 
-This project is [MIT](LICENSE.md) licensed.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- LICENSE -->
 
+## 📝 License <a name="license"></a>
 
-## 📝 License
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Also fix the broken link and add your own license and remove the placeholder text under the license section also